gnovm/tests/file_test.go

```diff
@@ -80,6 +80,19 @@ func runFileTests(t *testing.T, baseDir string, ignore []string, opts ...RunFile
 	for _, file := range files {
 		file := file
 		t.Run(file.Name(), func(t *testing.T) {
+			if file.Name() == "composite0b.gno" {
```

Comment:
Here, it's because TestChallenges was never called in the CI in the first place, and isn't meant to.

"challenges" is a directory of failing GnoVM tests, which should be passing. "But these should be tracked as issues rather than as sparce files", and I agree with you.

I can see a few approaches:

1. Place t.Skip by default on TestChallenges.
2.  "negate" TestChallenges (ie. try to catch a panic, and if there is no panic then error out on the CI -> notify the programmer when a "challenge" has been fixed)
3. Remove the challenges directory and simply have the challenges as issues here on GitHub.

Reply:
I'm all for 3️⃣ , it's much easier and cleaner to manage

cc @Kouteki

Reply:
3 sounds good to me.

Reply:
Adding this to the TODO list.